### PR TITLE
cairo-desktop: Add version 0.4.80

### PR DESCRIPTION
**Pull request by kevinboss**
https://github.com/cairoshell/cairoshell

I am not the owner of said project

### CODE REVIEW

File: bucket/cairo-shell.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.4.80",
+    "description": "Cairo Desktop Environment",
+    "homepage": "https://cairoshell.com/",
+    "license": "Apache 2.0",
```
Comment (Ash258) [inline bucket/cairo-shell.json:5]: Invalid SPDX identifier

File: bucket/cairo-shell.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.4.80",
+    "description": "Cairo Desktop Environment",
```
Comment (Ash258) [inline bucket/cairo-shell.json:3]: What does this mean?? Completely saying-nothing description for end-user

File: bucket/cairo-shell.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "homepage": "https://cairoshell.com/",
+    "license": "Apache 2.0",
+    "architecture": {
```
Comment (Ash258) [inline bucket/cairo-shell.json:6]: Add notes about .net framework requirements

File: bucket/cairo-shell.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.4.80",
+    "description": "Cairo Desktop Environment",
+    "homepage": "https://cairoshell.com/",
```
Comment (Ash258) [inline bucket/cairo-shell.json:4]:
```suggestion
    "homepage": "https://cairoshell.com",
```

File: bucket/cairo-shell.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    "bin": "CairoDesktop.exe",
+    "shortcuts": [
+        [
+            "CairoDesktop.exe",
+            "Cairo Desktop"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/cairoshell/cairoshell",
+        "regex": "tag/v([\\d.]+)"
```
Comment (Ash258) [inline bucket/cairo-shell.json:21]: Ideally find out winsparkle URL, which is used in the application

```suggestion
        "url": "https://github.com/cairoshell/cairoshell/releases",
        "regex": "download/v([\\d.]+)/"
```

File: bucket/cairo-shell.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.4.80",
+    "description": "Cairo Desktop Environment",
+    "homepage": "https://cairoshell.com/",
+    "license": "Apache 2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/cairoshell/cairoshell/releases/download/v0.4.80/CairoSetup_64bit.exe#/dl.7z",
+            "hash": "b7a37dc28ec7a78f0906d92843619e431513719051e0186153779b3cf31ef2fd"
+        }
```
Comment (Ash258) [inline bucket/cairo-shell.json:10]: Add 32bit version

File: bucket/cairo-shell.json [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    "bin": "CairoDesktop.exe",
```
Comment (Ash258) [inline bucket/cairo-shell.json:12]: What are available command line parameters?

File: bucket/cairo-shell.json [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+    ],
+    "checkver": {
+        "github": "https://github.com/cairoshell/cairoshell",
+        "regex": "tag/v([\\d.]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/cairoshell/cairoshell/releases/download/v$version/CairoSetup_64bit.exe#/dl.7z"
+            }
```
Comment (Ash258) [inline bucket/cairo-shell.json:27]: 32bit

File: bucket/cairo-shell.json [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+    "version": "0.4.80",
+    "description": "Cairo Desktop Environment",
+    "homepage": "https://cairoshell.com/",
+    "license": "Apache 2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/cairoshell/cairoshell/releases/download/v0.4.80/CairoSetup_64bit.exe#/dl.7z",
+            "hash": "b7a37dc28ec7a78f0906d92843619e431513719051e0186153779b3cf31ef2fd"
+        }
+    },
```
Comment (Ash258) [inline bucket/cairo-shell.json:11]: Remove `$PLUGINSDIR` folder from application directory in `pre_install`

**Review by Ash258 — CHANGES_REQUESTED** [review]
The application is not `cairo-shell`. The solution of project is named `Cairo Desktop`, download artifacts are `desktop`, executable is `desktop`

Does not make sense to name the manifest as it. It is `cairo-desktop`

File: bucket/cairo-shell.json (hunk 2 above)
Comment (kevinboss) [reply bucket/cairo-shell.json]: I updated the description according to other bucket descriptions I have seen.

File: bucket/cairo-shell.json (hunk 7 above)
Comment (kevinboss) [reply bucket/cairo-shell.json]: https://github.com/cairoshell/cairoshell/wiki/Startup-Arguments

I am not sure if it is sensible to make them public via shortcut as they are probably rarely needed and only clutter up the start menu

File: bucket/cairo-desktop.json [hunk 10]
```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.4.80",
+    "description": "Cairo is a alternative desktop environment for Windows.",
+    "homepage": "https://cairoshell.com",
+    "license": "Apache-2.0",
+    "notes": ".NET 4.7.1 is required https://www.microsoft.com/en-us/download/details.aspx?id=56115",
```
Comment (Ash258) [inline bucket/cairo-desktop.json:6]:
```suggestion
    "notes": ".NET 4.7.1 is required",
```

File: bucket/cairo-desktop.json [hunk 11]
```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.4.80",
+    "description": "Cairo is a alternative desktop environment for Windows.",
```
Comment (Ash258) [inline bucket/cairo-desktop.json:3]:
```suggestion
    "description": "Alternative desktop environment for Windows",
```

File: bucket/cairo-desktop.json [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+        "64bit": {
+            "url": "https://github.com/cairoshell/cairoshell/releases/download/v0.4.80/CairoSetup_64bit.exe#/dl.7z",
+            "hash": "b7a37dc28ec7a78f0906d92843619e431513719051e0186153779b3cf31ef2fd"
+        },
+        "32bit": {
+            "url": "https://github.com/cairoshell/cairoshell/releases/download/v0.4.80/CairoSetup_32bit.exe#/dl.7z",
+            "hash": "90c3461be314cb97674680eb000a749626cc9040cbf640089273cff597205589"
+        }
+    },
+    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\" -Recurse",
```
Comment (Ash258) [inline bucket/cairo-desktop.json:17]:
```suggestion
    "pre_install": "Remove-Item \"$dir\\`$*\" -Recurse",
```

File: bucket/cairo-desktop.json [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+    "bin": "CairoDesktop.exe",
+    "shortcuts": [
+        [
+            "CairoDesktop.exe",
+            "Cairo Desktop"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/cairoshell/cairoshell/releases",
+        "regex": "download/v([\\d.]+)/"
```
Comment (Ash258) [inline bucket/cairo-desktop.json:27]:
```suggestion
        "url": "https://cairoshell.github.io/appdescriptor.rss",
        "regex": ">Cairo Desktop\\s+([\\d.]+)</"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — APPROVED** [review]

**Pull request merged**